ballerina/service_endpoint.bal

```diff
@@ -46,7 +46,7 @@ public isolated class Listener {
     #
     # + return - An `error` if an error occurred during the listener stopping process or else `()`
     public isolated function gracefulStop() returns error? {
-        return;
+        return externGracefulStop(self);
```

Comment:
you can call the extern function right from here rather using `externGracefulStop()`